cypress/integration/Transfer/highlight-range-of-options/index.js

```diff
@@ -0,0 +1,325 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps'
+
+import { extractOptionFromElement } from '../common'
+
+Given('the option list has one or more items', () => {
```

Comment:
For the givens in this file that refer to a specific number of items/options, maybe we should assert that that's actually true. To ensure that the statement matches what's actually happening.